drivers/sensor/bmi08x/Kconfig

```diff
@@ -0,0 +1,226 @@
+
+config BMI08X_ACCEL_RANGE_4G_6G
+	bool "BMI085 4G / BMI088 6G"
+
+config BMI08X_ACCEL_RANGE_8G_12G
+	bool "BMI085 8G / BMI088 12G"
+
+config BMI08X_ACCEL_RANGE_16G_24G
+	bool "BMI085 16G / BMI088 24G"
+endchoice
+
+choice BMI08X_ACCEL_ODR
+	prompt "Accelerometer sampling frequency."
+	default BMI08X_ACCEL_ODR_1600 if BMI08X_DATA_SYNC_ODR_2000
+	default BMI08X_ACCEL_ODR_800 if BMI08X_DATA_SYNC_ODR_1000
+	default BMI08X_ACCEL_ODR_400 if BMI08X_DATA_SYNC_ODR_400
+	default BMI08X_ACCEL_ODR_RUNTIME
+
+config BMI08X_ACCEL_ODR_RUNTIME
+	bool "Set at runtime." if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_25_2
+	bool "12.5 Hz" if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_25
+	bool "25 Hz" if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_50
+	bool "50 Hz" if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_100
+	bool "100 Hz" if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_200
+	bool "200 Hz" if !BMI08X_DATA_SYNC
+
+config BMI08X_ACCEL_ODR_400
+	bool "400 Hz" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_400) || !BMI08X_DATA_SYNC)
+
+config BMI08X_ACCEL_ODR_800
+	bool "800 Hz" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_1000) || !BMI08X_DATA_SYNC)
+
+config BMI08X_ACCEL_ODR_1600
+	bool "1600 Hz" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_2000) || !BMI08X_DATA_SYNC)
+endchoice
+
+choice BMI08X_GYRO_RANGE
+	prompt "Gyroscope range setting."
+	default BMI08X_GYRO_RANGE_RUNTIME
+
+config BMI08X_GYRO_RANGE_RUNTIME
+	bool "Set at runtime."
+
+config BMI08X_GYRO_RANGE_2000DPS
+	bool "2000 DPS"
+
+config BMI08X_GYRO_RANGE_1000DPS
+	bool "1000 DPS"
+
+config BMI08X_GYRO_RANGE_500DPS
+	bool "500 DPS"
+
+config BMI08X_GYRO_RANGE_250DPS
+	bool "250 DPS"
+
+config BMI08X_GYRO_RANGE_125DPS
+	bool "125 DPS"
+endchoice
+
+choice BMI08X_GYRO_ODR
+	prompt "Gyroscope sampling frequency."
+	default BMI08X_GYRO_ODR_2000_532 if BMI08X_DATA_SYNC_ODR_2000
+	default BMI08X_GYRO_ODR_1000_116 if BMI08X_DATA_SYNC_ODR_1000
+	default BMI08X_GYRO_ODR_400_47 if BMI08X_DATA_SYNC_ODR_400
+	default BMI08X_GYRO_ODR_RUNTIME
+
+config BMI08X_GYRO_ODR_RUNTIME
+	bool "Set at runtime." if !BMI08X_DATA_SYNC
+
+config BMI08X_GYRO_ODR_100_12
+	bool "100 Hz, 12Hz filter" if !BMI08X_DATA_SYNC
+
+config BMI08X_GYRO_ODR_100_32
+	bool "100 Hz, 32Hz filter" if !BMI08X_DATA_SYNC
+
+config BMI08X_GYRO_ODR_200_23
+	bool "200 Hz, 23Hz filter" if !BMI08X_DATA_SYNC
+
+config BMI08X_GYRO_ODR_200_64
+	bool "200 Hz, 64Hz filter" if !BMI08X_DATA_SYNC
+
+config BMI08X_GYRO_ODR_400_47
+	bool "400 Hz, 47Hz filter" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_400) || !BMI08X_DATA_SYNC)
+
+config BMI08X_GYRO_ODR_1000_116
+	bool "1000 Hz, 116Hz filter" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_1000) || !BMI08X_DATA_SYNC)
+
+config BMI08X_GYRO_ODR_2000_230
+	bool "2000 Hz, 230Hz filter" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_2000) || !BMI08X_DATA_SYNC)
+
+config BMI08X_GYRO_ODR_2000_532
+	bool "2000 Hz, 532Hz filter" if ((BMI08X_DATA_SYNC && BMI08X_DATA_SYNC_ODR_2000) || !BMI08X_DATA_SYNC)
+endchoice
```

Comment:
Move these to devicetree properties to allow per-instance configuration

Reply:
not fixed yet in the last push, but will get to

Reply:
done